### DISCUSSION TITLE
Add member debugging to Debug Tools

### DISCUSSION THREAD

**Discussion by robinsowell** [ExpressionEngine/ExpressionEngine#3024] [Feature Requests]
You can debug Templates and Field Types in 'Utilities', but sometimes member data gets off too.  

For example, just had someone getting blanks and errors in the author filter on the entry page.  They had entries authored by member ids that no longer exist.  I've seen it happen with Templates as well- errors because last author no longer exists.  This shouldn't happen, but would be nice and fairly easy to have a way for folks to run a check, make sure all of the member data is structurally sound and replace any missing values in relationships.

I also get requests for ways to prune inactive members (though maybe that belongs in the actual member section).

**Comment by intoeetive** [comment]
Missing last_author_id for templates does not seem to be in issue in EE 7.3.13

The missing entry author problem is still valid issue however, I am adding a check for it among others in https://github.com/ExpressionEngine/ExpressionEngine/pull/3944
